scripts/build.py

```diff
@@ -924,6 +932,12 @@ def archive_engine(self):
             gdc_bin = join(bin_dir, gdc_name)
             self.upload_to_archive(gdc_bin, '%s/%s' % (full_archive_path, gdc_name))
 
+        # upload libdlib_shared on desktop platforms
+        if self.target_platform in ['x86_64-linux', 'x86_64-darwin', 'x86_64-win32']:
+            lib_name = format_lib("dlib_shared", self.target_platform)[0]
+            lib_bin = join(bin_dir, lib_name)
+            self.upload_to_archive(lib_bin, '%s/%s' % (full_archive_path, lib_name))
```

Comment:
In order to easily make a slim Defold SDK for testing locally, it's good if there is a built dlib_shared available from d.defold.com that you can easily download.